prep-html-links/contact-us/contact-us.html

```diff
@@ -8,7 +8,7 @@
 <body>
   <h1>Contact Us</h1>
   <div>
-    <a href="http://127.0.0.1:5500/prep-html-links/index.html">Click here to go to the home page</a>
+    <a href="/prep-html-links/index.html">Click here to go to the home page</a>
```

Comment:
We are getting much closer 👍 . Here we want to use a relative path `href="../index.html"` so that these files don't need to necessarily be inside of a `prep-html-links` directory to still link properly.